README.md

```diff
@@ -55,6 +55,10 @@ Before we get into the details, we'd like to outline our goals and our guiding p
     dotnet run figlet doc            // prints the documentation of the figlet module
     dotnet run figlet example        // shows examples of the module in action
     ```
+   
+1. **Bonus: Do the same for Dumpify**
+
+   Along the way we also added a separate module wrapping the functionality of Dumpify following the same principles. While the main focus remains with Spectre.Console (hence the name of the package), we do think Dumpify's capabilities are useful and related enough to just slap it on top of our package.
```

Comment:
What's up with this whitespace?

Reply:
Maybe also mention tha Dumpify internally already uses Spectre.Console?

Reply:
If you mean the leading one, it is consistent with the other sections: 

<img width="686" alt="image" src="https://github.com/EluciusFTW/SpectreCoff/assets/17160319/7d2bd4f8-740f-4183-b779-c73ccc0c1203">

I'll either leave it as is, or I will remove them everywhere.

Reply:
Done, please resolve if you can live with this or tell me to change this (or the brief mention of Spectre) otherwise :)